src/LabelImporter.php

```diff
@@ -12,32 +10,22 @@ class LabelImporter
      */
     public function importLabels(\CultureFeed_Cdb_Item_Base $item, $jsonLD)
     {
-        /** @var CultureFeed_Cdb_Data_Keyword[] $keywords */
-        $keywords = array_values($item->getKeywords(true));
-
-        $validKeywords = array_filter(
-            $keywords,
-            function (CultureFeed_Cdb_Data_Keyword $keyword) {
-                return strlen(trim($keyword->getValue())) > 0;
-            }
+        $labelCollection = LabelCollection::fromKeywords(
+            $item->getKeywords(true)
         );
 
-        $visibleKeywords = array_filter(
-            $validKeywords,
-            function (CultureFeed_Cdb_Data_Keyword $keyword) {
-                return $keyword->isVisible();
+        $visibleLabelCollection = new LabelCollection();
+        $hiddenLabelCollection = new LabelCollection();
+        foreach ($labelCollection->asArray() as $label) {
+            if ($label->isVisible()) {
+                $visibleLabelCollection = $visibleLabelCollection->with($label);
+            } else {
+                $hiddenLabelCollection = $hiddenLabelCollection->with($label);
```

Comment:
Can we move this as new methods into the LabelCollection class? E.g. LabelCollection::onlyVisible() and LabelCollection::onlyInvisible(), which both return a new LabelCollection?